### PR TITLE
Foundation Assessment  2

### PR DESCRIPTION
**Pull request by gurtatiLND**

### CODE REVIEW

File: foundation_assessment_ii.sql [hunk 1]
```diff
@@ -0,0 +1,93 @@
...
+    Four_K BOOLEAN
+);
+
+CREATE TABLE showings (
+    Showing_ID INT PRIMARY KEY AUTO_INCREMENT,
+    Movie_ID INTEGER NOT NULL,
+    Screen_ID INTEGER NOT NULL,
+    Start_Time time,
+    Available_Seats INTEGER NOT NULL
+);
```
Comment (EslemOuederni) [inline foundation_assessment_ii.sql:22]: **10/10**
Awesome !

File: foundation_assessment_ii.sql [hunk 2]
```diff
@@ -0,0 +1,93 @@
...
+-- Display the results in time order.
+
+SELECT m.movie_name,
+       s.start_time
+FROM movie_info as m
+RIGHT JOIN showings as s
+ON m.movie_id = s.movie_id
+WHERE s.start_time >= '12:00:00'
+AND s.available_seats > 1
+ORDEr BY s.start_time;
```
Comment (EslemOuederni) [inline foundation_assessment_ii.sql:85]: **5/6**
the question says : _all movies that play after 12:00 given there is at least 1 available seat._ 
after mean s.start_time > 12:00:00 and available_seats >0 (or you can do >=1)

File: foundation_assessment_ii.sql [hunk 3]
```diff
@@ -0,0 +1,93 @@
...
+
+-- Return the name of the movie with the most showings.
```
Comment (EslemOuederni) [inline foundation_assessment_ii.sql:87]: make sure to at least add select phrase if you can't think of a solution (select the right info would have given you 2 points)

File: isogram.py [hunk 4]
```diff
@@ -0,0 +1,19 @@
+# write function that takes in an input and checks to see if it’s an isogram.
```
Comment (EslemOuederni) [inline isogram.py:1]: **7/7**

can you explain the space and hyph that you added?

File: min_classes.py
Comment (EslemOuederni) [inline min_classes.py:1]: **7.5/25**

This can be further improved. 
You're going on the right way, except I'm not sure of using floor method here.

File: test_isogram.py
Comment (EslemOuederni) [inline test_isogram.py:1]: **10/12**
when it comes to testing, you should cover valid, invalid and edge cases. Of course you can create as many invalid or invalid tests. For the exam we either specify how many tests or you would ideally create these three ones.
upper_letters can be thought of as edge case, and test_if_number will be considered as an invalid since the result should be false.

File: theory_questions_2.MD [hunk 5]
```diff
@@ -1,64 +1,139 @@
...
+
+
+### Section 1: Theory Questions [31 marks]
+
+
+
+| 1.1 What does SDLC stand for? | 1 mark |
+| ----------- | ----------- |
 
 The SDLC is a software development life cycle, which means it’s a process or methodology which helps create high-quality software in the shortest time with the lowest amount of money.
```
Comment (EslemOuederni) [inline theory_questions_2.MD:35]: you don't need to give explanation for this question

File: theory_questions_2.MD [hunk 6]
```diff
@@ -1,64 +1,139 @@
...
 
 
-**1.6**
 
-*Name 2 debugging methods, and when you would use them.*
+| 1.6   Name 2 debugging methods, and when you would use them. | 4 marks |
+| ----------- | ----------- |
 
--Breakpoint() - this is a built - in debugging function which is new in Python 3.7 and it’s equivalent to the older 
+- Breakpoint() - this is a built - in debugging function which is new in Python 3.7 and it’s equivalent to the older
```
Comment (EslemOuederni) [inline theory_questions_2.MD:71]: **2/4**
here we should mention something that we've used in class, like print()

File: theory_questions_2.MD [hunk 7]
```diff
@@ -1,64 +1,139 @@
...
+B: when the user click on “order delivery”: 
+   - see all options of meals
+   - click and add meal to the busket
+   - then click on the bucket and made a checkout
+   - choose where to deliver 
+   - confirm the total price for food and delivery
+   - choose paymant method and pay
+- Get notifications from restaurant about driver and his estimated time, 
+or about readiness of the order
+- ask a review from client few hours later
```
Comment (EslemOuederni) [inline theory_questions_2.MD:124]: **2/3**
- see all options of meals is a requirement of course 
- Get notifications from restaurant about driver and his estimated time, 
You're more or less mentioning the process of the ordering and not the requirements. Basic requirements can be something like : available items should be up to date

File: theory_questions_2.MD
Comment (EslemOuederni) [inline theory_questions_2.MD:1]: **28/31**
Awesome

**Review by EslemOuederni — APPROVED** [review]
:rocket: Total : **68/100**
**28/31: Theory Questions 
17/19:concept questions 
7.5/25: Python challenge** 
**15/25: SQL** 
_Full feedback will be emailed_

Awesome job ! You can always comment on the GitHub (pull request discussion) if you have any questions.

File: isogram.py (hunk 4 above)
Comment (gurtatiLND) [reply isogram.py:1]: Just thought that this is like an option could be.
